proto/tic-tac-toe.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+service Game {
+    rpc Connect (stream Request) return (stream Responce);
+}
+
+message Request {
+    string type = 1;
+    repeated int8 payload = 2;
```

Comment:
`repeated int8` must be `bytes`
see more at [Scalar Value Types](https://developers.google.com/protocol-buffers/docs/proto3#scalar)